### PR TITLE
Update basic-skill.md

### PR DESCRIPTION
**Pull request by immature-mono**
# Description
Small wording changes to improve readability. 

## Status
**READY**

## Type of change

- Documentation (fix or adds documentation)

# How Has This Been Tested?

- Tested for readability

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation (if applicable)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: docs/tutorials/basic-skill.md [hunk 1]
```diff
@@ -82,21 +82,21 @@ class MySkill(Skill):
         await message.respond('Good thanks! My load average is 0.2, 0.1, 0.1.')
 ```
 
-Our skill is done and opsdroid will be able to answer like in the [Opsdroid main page](https://opsdroid.github.io). The next thing that is left to do is to add the skill to the opsdroid configuration file.
+Our skill is done and opsdroid will be able to respond like in the [Opsdroid main page](https://opsdroid.github.io). The final thing  left to do is to add the skill to the opsdroid configuration file.
```
Comment (jacobtomlinson) [inline docs/tutorials/basic-skill.md]:
```suggestion
Our skill is done and opsdroid will be able to respond like in the [Opsdroid main page](https://opsdroid.github.io). The final thing left to do is to add the skill to the opsdroid configuration file.
```

Comment (immature-mono) [reply docs/tutorials/basic-skill.md]: Thanks for that, I removed the extra space :)

**Comment by jacobtomlinson** [comment]
I'm afraid since raising this someone else has contributed changes to this file resulting in conflicts. Could you please resolve these.

**Comment by jacobtomlinson** [comment]
Good stuff!

We love to thank contributors by sending them [opsdroid stickers](https://medium.com/opsdroid/contributor-sticker-packs-738058ceda59)! Fill in the [claim form](https://forms.gle/iQZV9DjkaUiSaDXK6) to get yours!

**Pull request merged**